### DISCUSSION TITLE
No such module 'AppKit'

### DISCUSSION THREAD

**Discussion by Manilko** [SwiftGen/SwiftGen#1116] [Q&A]
I connected SwiftGen via spm, and and got an error  >> No such module 'AppKit'
SwiftGenKit  // Copyright © 2022 SwiftGen  // MIT Licence
there are three imported frameworks import AppKit  import Foundation.  import PathKit

**Comment by djbe** [comment]
Please check https://github.com/SwiftGen/SwiftGenPlugin, do not add SwiftGen itself as a SPM dependency.
